File: 1785.cpp
#include <iostream>

using namespace std;

int main()
{
 int number;
 cin>>number;
 if(number<5)
  cout<<"few";
 else if(number<10)
  cout<<"several";
 else if(number<20)
  cout<<"pack";
 else if(number<50)
  cout<<"lots";
 else if(number<100)
  cout<<"horde";
 else if(number<250)
  cout<<"throng";
 else if(number<500)
  cout<<"swarm";
 else if(number<1000)
  cout<<"zounds";
 else
  cout<<"legion";
 return 0;
}

File: 1785.cs
using System;

namespace flx3sem
{
    class Program
    {
        static void Main(string[] args)
        {
            int a;
            var s = Console.ReadLine();
            a = int.Parse(s);
            if ((a > 0) && (a < 5)) Console.WriteLine("few");
            else if (a < 10) Console.WriteLine("several");
            else if (a < 20) Console.WriteLine("pack");
            else if (a < 50) Console.WriteLine("lots");
            else if (a < 100) Console.WriteLine("horde");
            else if (a < 250) Console.WriteLine("throng");
            else if (a < 500) Console.WriteLine("swarm");
            else if (a < 1000) Console.WriteLine("zounds");
            else Console.WriteLine("legion");
        }
    }
}
